src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
-public class MainApp {
+public static void main(String[] args) {
+Machine[] machines = {
+        new Truck(),
+        new Bulldozer(),
+        new Excavator()
+};
 
+for (Machine machine : machines){
+    machine.doWork();
+    machine.stopWork();
+}
+    }
```

Comment:
It seems that the `main` method is not enclosed within a class definition. According to Java syntax, methods should be part of a class or an interface. Please ensure that the `main` method is correctly placed inside a class, typically named `MainApp` or similar.